src/Tribe/Views/V2/View.php

```diff
@@ -1146,7 +1146,19 @@ protected function setup_repository_args( Context $context = null ) {
 
 		// Set's up category URL for all views.
 		if ( ! empty( $context_arr[ TEC::TAXONOMY ] ) ) {
-			$args[ TEC::TAXONOMY ] = $context_arr[ TEC::TAXONOMY ];
+			if ( ! is_array( $context_arr[ TEC::TAXONOMY ] ) ) {
+				$args[ TEC::TAXONOMY ] = $context_arr[ TEC::TAXONOMY ];
+			} else {
+				if ( empty( $args['tax_query'] ) ) {
+					$args['tax_query'] = [];
```

Comment:
:warning: Detected usage of tax_query, possible slow query.

<sub>hash: 314240cda0ff6d743adbbc155a93da12</sub>